php/account-details.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+    require_once('lib/MailChimp.php');
+
+    if (file_exists('../APIKEY')) {
```

Comment:
Put '../APIKEY' into a variable since the same string is repeated in the next line.